techniques/android/MASTG-TECH-0005.md

```diff
@@ -3,10 +3,97 @@ title: Installing Apps
 platform: android
 ---
 
-Use `adb install` to install an APK on an emulator or connected device.
+## Basic APK installation
```

Comment:
could you please fix all other title capitalizations?

https://mas.owasp.org/contributing/5_Style_Guide/#title-capitalization

```suggestion
## Basic APK Installation
```